goss/vars.yaml

```diff
@@ -0,0 +1,16 @@
+model_name: cos-model
+
+enable_self_tests: true
+enable_relation_tests: true
+enable_post_relation_tests: true
+enable_network_tests: true
+enable_k8s_tests: true
+
+alertmanager_app_name: alertmanager
+catalogue_app_name: catalogue
+grafana_app_name: grafana
+loki_app_name: loki
+prometheus_app_name: prometheus
+traefik_app_name: traefik
+
+pebble_checks: [alertmanager, catalogue, grafana, loki, prometheus, traefik]
```

Comment:
For the purpose of this example, should we:
1. Name this file `cos.yaml`? To support multiple teams, use subfolders per app or per solution?
2. Rename the `pebble_checks`  key to `apps`?
3. Use a nested structure such as:

```yaml
apps:
  alertmanager:
    name: alertmanager
  loki:
    name: loki
  # etc.
```

Reply:
I included using a nested structure in the vars.yaml. I do not think naming it cos.yaml makes it easier to understand and making subfolders per solution makes more sense (even for contributions). However, this repo is specific to cos. It only makes sense to make subfolders for cos related deployments like `$root/goss/cos-edge`, ...